addon/helpers/to-money.js

```diff
@@ -8,16 +8,15 @@ const DEFAULT_LOCALE = 'en-gb';
 export default Helper.extend({
   i18n: service('i18n'),
 
-  compute([params]) {
-    let i18n = this.get('i18n');
-
+  compute([params], namedArgs) {
     if (isEmpty(params)) {
       return '';
     }
 
-    let locale = i18n.get('locale') || DEFAULT_LOCALE;
-    let { amount, currency_code } = params;
+    const locale = this.get('i18n.locale') || DEFAULT_LOCALE;
+    const { amount, currency_code } = params;
+    const transformedAmount = isPresent(namedArgs.transform) && !namedArgs.transform ? amount * 100 : amount;
```

Comment:
Completely agree, but that wouldn't be backwards compatible. There are some cases where we use the util directly (that's why we have the util), and it would break every occurence of its use